submissions/AndriiZahumnyi/js-dom/script/index.js

```diff
@@ -0,0 +1,27 @@
+
+const hamburgerToggle = document.querySelector('#menu-toggle');
+
+const flexMenu = document.querySelector('.flex-menu');
+
+const arrAside = [".aside-html", ".aside-css", ".aside-bootstrap", ".aside-js", ".aside-jquery", ".aside-git"];
+
+const arrContent = [".html-section", ".css-section", ".bootstrap-section", ".js-section", ".jquery-section", ".git-section"];
+
+const asideMenuList = document.getElementById("aside-menu-list");
+
+const lengthOfArrAside = arrAside.length;
```

Comment:
I think now we don't need `arrAside`, `arrContent` and `lengthOfArrAside` constants. You can remove them